### DISCUSSION TITLE
Nested sidebar renders duplicate links

### DISCUSSION THREAD

**Discussion by erondpowell** [vuejs/vitepress#1275] [Q&A] [answered]
In our folder project, we are trying to use the nested sidebar feature.
- Each parent is a link to a page. ex: `/configuration/data-model.md`
- Each nested page is in a folder. ex: `/configuration/data-model/relationships.md`

Everything works fine in dev, but on prod testing, nested links are rendered multiple times.
We are not sure if it is a misconfiguration on our end, or a bug in the sidebar:
https://github.com/directus/docs/pull/109

PS if its a misconfiguration, I'm very very happy to contribute a docs PR for Vitepress, if interested.

Thanks 🙏

**Comment by brc-dd** [comment, marked as answer]
fixed in #1279
